hazelcast/src/main/java/com/hazelcast/jet/core/test/TestProcessorMetaSupplierContext.java

```diff
@@ -173,6 +173,11 @@ public ProcessingGuarantee processingGuarantee() {
         return processingGuarantee;
     }
 
+    @Override
+    public long maxProcessorAccumulatedRecords() {
+        return Long.MAX_VALUE;
```

Comment:
We should have a field here that the user can set. One can then test processors with a desired value.